model/textparse/openmetricsparse.go

```diff
@@ -269,6 +343,7 @@ func (p *OpenMetricsParser) Next() (Entry, error) {
 				mEnd--
 			}
 			p.offsets = append(p.offsets, mStart, mEnd)
+			p.mName = string(p.l.b[mStart:mEnd])
```

Comment:
IMO we don't need this.

Reply:
rationale explained in above comment

Reply:
Thanks! Responded, it's not needed it seems?

Reply:
removed